codes/dart/chapter_array_and_linkedlist/my_list.dart

```diff
@@ -0,0 +1,135 @@
+/**
+ * File: my_list.dart
+ * Created Time: 2023-02-05
+ * Author: Jefferson (JeffersonHuang77@gmail.com)
+ */
+class MyList {
+  /// [_nums]  数组（存储列表元素）
+  late List<int> _nums;
+
+  /// [_capacity]列表容量
+  int _capacity = 10;
```

Comment:
Can we use 4 spaces for indentation in Dart? If so, please use 4 spaces to keep the consistency with other languages.

Reply:
Yes, 4 spaces can be used, but this is against the dart style specification. Are you sure you need to replace?

Reply:
No, let us keep 2 spaces.